docs/en/latest/config.json

```diff
@@ -106,7 +106,8 @@
             "plugins/traffic-split",
             "plugins/request-id",
             "plugins/proxy-control",
-            "plugins/client-control"
+            "plugins/client-control",
+            "plugins/workflow"
```

Comment:
We should put this in the same category of serverless-pre/post plugin.

Reply:
done